### PR TITLE
go: sqle/cluster: mysqldb_persister.go: Add exponential backoff to users and grants replication to avoid spamming logs.

### PR DESCRIPTION
**Pull request by reltuk**

### CODE REVIEW

**Review by timsehn — CHANGES_REQUESTED** [review]
Jokes.

File: go/libraries/doltcore/sqle/cluster/controller.go [hunk 1]
```diff
@@ -165,9 +166,14 @@ func NewController(lgr *logrus.Logger, cfg Config, pCfg config.ReadWriteConfig)
 	}
 	ret.mysqlDbReplicas = make([]*mysqlDbReplica, len(clients))
 	for i := range ret.mysqlDbReplicas {
+		bo := backoff.NewExponentialBackOff()
+		bo.InitialInterval = time.Second
+		bo.MaxInterval = time.Minute
+		bo.MaxElapsedTime = 0
```
Comment (timsehn) [inline go/libraries/doltcore/sqle/cluster/controller.go:172]: Jitter please. Google is sad.

**Review by timsehn — CHANGES_REQUESTED** [review]
What? No jitter!

File: go/libraries/doltcore/sqle/cluster/mysqldb_persister.go [hunk 2]
```diff
@@ -49,6 +50,7 @@ type mysqlDbReplica struct {
 	version  uint32
 
 	replicatedVersion uint32
+	backoff           backoff.BackOff
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/cluster/mysqldb_persister.go:53]: back off, man

back off, i said Back off!

**Review by timsehn — APPROVED** [review]

**Review by zachmu — APPROVED** [review]
LGTM!

**Pull request merged**